specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/examples/createOrUpdateDynamicMetricAlertSingleResource.json

```diff
@@ -0,0 +1,109 @@
+{
+    "parameters": {
+      "subscriptionId": "14ddf0c5-77c5-4b53-84f6-e1fa43ad68f7",
+      "resourceGroupName": "gigtest",
+      "ruleName": "chiricutin",
+      "api-version": "2018-03-01",
+      "parameters": {
+        "location": "global",
+        "tags": {},
+        "properties": {
+          "description": "This is the description of the rule1",
+          "severity": 3,
+          "enabled": true,
+          "scopes": [
+            "/subscriptions/14ddf0c5-77c5-4b53-84f6-e1fa43ad68f7/resourceGroups/gigtest/providers/Microsoft.Compute/virtualMachines/gigwadme"
```

Comment:
@meday89 if you are using real subscriptionId, use a fake subscriptionId.
We recommend (000000-0000-00000...…..)

Reply:
Fixed, please see updates.